ambry-api/src/main/java/com/github/ambry/config/MySqlNamedBlobDbConfig.java

```diff
@@ -58,14 +58,14 @@ public class MySqlNamedBlobDbConfig {
    * The maximum number of entries to return per each stale blobs pull request.
    */
   @Config(QUERY_STALE_DATA_MAX_RESULTS)
-  @Default("1000")
+  @Default("10000")
```

Comment:
why increase the result, can we keep it as 1000, this would limit the time to run the query.

Reply:
I thought 1000 is too few to handle all stale blobs in one day (since i always schedule the cleanup to run once per day). 
Now i am using config-driven schedule interval for the named blob cleanup process, I just changed it back to 1000.